packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -122,7 +125,9 @@ export class StripeAutomaticTaxConverter {
       const taxEnabled = await this.enableTaxForCustomer(customer);
       if (!taxEnabled) return; // Do not enable tax if customer is not taxable
 
-      await this.enableTaxForSubscription(subscriptionId);
+      if (this.isEligibleSubscriptionProduct(product.id)) {
+        await this.enableTaxForSubscription(subscriptionId);
+      }
```

Comment:
If the subscription is FPN this should return so as not to generate a report entry.

Reply:
If the customer is still being updated, should they not still be in the report?

Reply:
The customer shouldn't be updated in that case - I missed that this clause comes after the customer update call. It should exit prior to updating the customer.

Reply:
Ah OK great thank you. I'll make the change.